mail_tracking/models/mail_message.py

```diff
@@ -63,35 +98,51 @@ def tracking_status(self):
             partners -= partners_already
             for partner in partners:
                 # If there is partners not included, then status is 'unknown'
-                partner_trackings.append((
-                    'unknown', False, partner.name, partner.id, partner.email))
+                # Because can be an Cc recipient
+                isCc = False
+                if partner.email in email_cc_list:
+                    email_cc_list.discard(partner.email)
+                    isCc = True
+                partner_trackings.append({
+                    'status': 'unknown',
+                    'status_human':
+                        self._partner_tracking_status_human_get('unknown'),
+                    'error_type': False,
+                    'error_description': False,
+                    'tracking_id': False,
+                    'recipient': partner.name,
+                    'partner_id': partner.id,
+                    'isCc': isCc,
+                })
+            for email in email_cc_list:
+                # If there is Cc without partner
+                partner_trackings.append({
+                    'status': 'unknown',
+                    'status_human':
+                        self._partner_tracking_status_human_get('unknown'),
+                    'error_type': False,
+                    'error_description': False,
+                    'tracking_id': False,
+                    'recipient': email,
+                    'partner_id': False,
+                    'isCc': True,
+                })
             res[message.id] = partner_trackings
         return res
 
     @api.multi
-    def _get_email_cc(self):
-        """This method gets all Cc mails and the associated partner if exist.
-            The result is a dictionary by 'message id' with a list of tuples
-            (str:email_cc, list:[partner id, partner display_name] or False)
-        """
+    def _get_failed_message(self):
```

Comment:
As this is multi, I think this name should change to plural:

```suggestion
    def _get_failed_messages(self):
```